mrtt-ui/src/components/ProjectDetailsForm.js

```diff
@@ -15,12 +10,19 @@ import {
 import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns'
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider'
 import { MobileDatePicker } from '@mui/x-date-pickers/MobileDatePicker'
+import { useForm, Controller } from 'react-hook-form'
+import { useState } from 'react'
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as Yup from 'yup'
 import Autocomplete from '@mui/material/Autocomplete'
+import axios from 'axios'
 
-import { MainFormDiv, FormQuestionDiv, SectionFormTitle } from '../styles/forms'
-import countries from '../data/countries.json'
+import { ErrorText } from '../styles/typography'
+import { MainFormDiv, FormQuestionDiv, SectionFormTitle, Form } from '../styles/forms'
 import { questionMapping } from '../data/questionMapping'
 import ButtonSubmit from './ButtonSubmit'
+import countries from '../data/countries.json'
+import language from '../language'
```

Comment:
I dont care if others alphabetically sort imports and other things, but when I touch code, I find myself compulsively doing it (sorry if its annoying in the diff)